### ISSUE TITLE
Add all repo projects to projects page

### ISSUE THREAD

**Issue by nikolajbaer** [opensandiego/opensandiego.github.io#35]
The projects page currently needs to have explicit projects listed in the folder.

Github provides all repo data in their "meta" object "site", that can be used to list out all projects.

https://help.github.com/en/articles/repository-metadata-on-github-pages

The idea here would be to have all repos listed (or maybe exclude archived ones), but only keep projects explicitly described as "featured" and "active".

In other words, if a project has a YAML file, it is active, otherwise we just list it as an inactive or previous project.

**Comment by nikolajbaer** [comment]
Closing this, instead we want a curated list.
